frontends/common/parser_options.h

```diff
@@ -57,6 +55,15 @@ class ParserOptions : public Util::Options {
     ParserOptions();
     std::vector<const char *> *process(int argc, char *const argv[]) override;
     enum class FrontendVersion { P4_14, P4_16 };
+
+    /// Records the result of the preprocessor.
+    struct PreprocesserResult {
```

Comment:
Probably a typo (Preproces_**e**_r)?

```suggestion
    struct PreprocessorResult {